portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,7 +29,17 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
-  private static ArrayList<String> comments = new ArrayList(Arrays.asList("Hi there!", "How are you?", "Howdy"));
+  private static ArrayList<String> comments = new ArrayList();
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String name = request.getParameter("name-input");
```

Comment:
If you aren't using name yet then you can remove this and add when you actually use it.